connect/api/src/test/java/org/apache/kafka/connect/data/SchemaProjectorTest.java

```diff
@@ -468,6 +468,21 @@ public void testMaybeCompatible() throws Exception {
         } catch (SchemaProjectorException e) {
             // expected
         }
+
+        Map<String, String> sourceEnumParamters = new HashMap<String, String>();
+        sourceEnumParamters.put("io.confluent.connect.avro.Enum", "FooEnum");
+        sourceEnumParamters.put("io.confluent.connect.avro.Enum.FooSymbol", "FooSymbol");
+        Schema sourceWithEnumParameters = SchemaBuilder.string().parameters(sourceEnumParamters);
+
+        Map<String, String> targetEnumParamters = new HashMap<String, String>();
+        targetEnumParamters.put("io.confluent.connect.avro.Enum", "FooEnum");
+        targetEnumParamters.put("io.confluent.connect.avro.Enum.FooSymbol", "FooSymbol");
+        targetEnumParamters.put("io.confluent.connect.avro.Enum.BarSymbol", "BarSymbol");
+        Schema targetWithEnumParameters = SchemaBuilder.string().parameters(targetEnumParamters);
+
+        Object projected = SchemaProjector.project(sourceWithEnumParameters, "FooSymbol", targetWithEnumParameters);
+
+        assertEquals("FooSymbol", projected);
```

Comment:
Would be nice to cover the following scenarios:
- Append enum symbol at the end (covered already)
- Add enum symbol in the middle
- Remove enum symbol 
- Update enum order
- Update enum doc

Reply:
👍

Reply:
@johnnycaol At this point in the kafka connect workflow we are only dealing with string types, so different permutations of the parameters will make no difference. Those scenarios would be good to have in an integration test but I don't think that creating new integration tests is in scope.